hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1843,6 +1901,7 @@ private VersionedFileStatus(
               hasAcl, false, false);
 
       this.version = version;
+      this.encryptionContext = encryptionContext;
```

Comment:
Inline with https://github.com/apache/hadoop/pull/3440#discussion_r989075168. Need to have encryptionContext in the VersionedFileStatus object.